content/javascript/concepts/strings/terms/replaceAll/replaceAll.md

```diff
@@ -37,3 +37,26 @@ This example results in the following output:
 ```shell
 I laugh, you laugh, we all laugh for ice cream.
 ```
+
+## Codebyte Example
+
+The first codebyte example of using `.replaceAll()` method is runnable, and it doesn't use [regular expressions (RegExp)](https://www.codecademy.com/resources/docs/javascript/regexp):
+
+```codebyte/javascript
+"AbCD_AbCD".replaceAll("b","B");
+// 'ABCD_ABCD'
+```
+
+The second codebyte example of using `.replaceAll()` method is runnable, and it uses [regular expressions (RegExp)](https://www.codecademy.com/resources/docs/javascript/regexp). Be careful that when using a regular expression search value, it must be global:
+
+```codebyte/javascript
+"AbCD_AbCD".replaceAll(/b/g,"B");
+// 'ABCD_ABCD'
+```
+
+This won't work:
+
+```codebyte/javascript
+"AbCD_AbCD".replaceAll(/b/,"B");
+// TypeError: replaceAll must be called with a global RegExp
```

Comment:
let's shift this comment to above line where you have added this wont work. Lets make it like `This won't work because and then the comment`.